spec/tree-view-spec.coffee

```diff
@@ -1105,6 +1119,39 @@ describe "TreeView", ->
             expect(atom.views.getView(pane)).toHaveFocus()
             expect(item.getPath()).toBe atom.project.getDirectories()[0].resolve('tree-view.txt')
 
+  xdescribe "opening existing opened files in existing split panes", ->
```

Comment:
It takes a long time, and probably isn't necessary. However, i wanted feedback on it. Sorry, i should have mentioned it.